modules/users/tests/server/user.server.model.tests.js

```diff
@@ -345,36 +345,37 @@ describe('User Model Unit Tests:', function () {
       
     });
 
-    it('should not allow double dotted characters in email address - "abcdef@abc..com"', function (done) {
```

Comment:
@mleanos assuming you were referring to the new code added on line 348, try and change it as follows:
```
    it('should allow single quote characters in email address - "abc\'def@abc.com"', function (done) {
      var _user1 = new User(user1);

      _user1.email = "abc\'def@abc.com";
      _user1.save(function (err) {
        err = {};
        if (!err) {
          _user1.remove(function (err_remove) {
            console.log('if');
            should.not.exist(err);
            should.not.exist(err_remove);
            done();
          });
        } else {
          console.log('else');
          should.not.exist(err);
          done();
        }
      });
    });
```

and then just play with the `err = {};` line and change it to null and back to an empty object to see how the tests behave.

The reason that `should.not.exist(err)` assertion is in the else block on 360 is that if the _user1.save operation fails and produces an err object then the assertion there on line 360 will fail the test because it expects err not to exist but it does.

Hopefully I explained it ok